components/tekton/pipelines/run-vse-sync-testsuite/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+commonAnnotations:
```

Comment:
I didn't know this feature exist, nice!
Please note that tekton CR is ignored by Argo CD in case you wanted to manage this with Argo https://github.com/redhat-partner-solutions/vse-catalog/blob/ac75321a40a9692a517629a3d411369288d6d6c0/components/argocd/base/openshift-gitops-argocd.yaml#L90-L97

Reply:
Isn't that just the `TaskRun` and `PipelineRun`? Or am I miss-understanding something?

Reply:
Yes you are correct, I thought you added this also for the `PipelineRun `...